packages/management/addons-manager-v2/kbld-config.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kbld.k14s.io/v1alpha1
```

Comment:
Same comment here as above. ```addons-manager-v2``` has changed to ```addons-manager```in [this PR](https://github.com/vmware-tanzu/tanzu-framework/pull/2315) . Please make appropriate changes. cc: @shyaamsn

Reply:
Updated.